flimfitgen.py

```diff
@@ -34,8 +35,8 @@ def usage():
     "/MCR_R2015b_maci64_installer.zip"
 
 for x, y in (
-        ("FLIMFIT_51_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
-        ("FLIMFIT_51_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
+        ("FLIMFIT_52_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
+        ("FLIMFIT_52_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
```

Comment:
I think we were discussing changing this link but then it was pointed out that the wIndows installer could also change.
The two changes aren't necessarily linked